.husky/commit-msg

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-BRANCH_NAME=$(git branch | grep '*' | sed 's/* //')
+BRANCH_NAME=$(git branch --no-color | grep --color=never '*' | sed 's/* //')
```

Comment:
Looks like my `git branch` solution is not the correct one. While scripting with Git plumbing commands should be used and `git branch` is not a plumbing but porcelain command.
https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain
https://git-blame.blogspot.com/2013/06/checking-current-branch-programatically.html

TLDR: `git symbolic-ref --short -q HEAD`

Reply:
ok! maybe that one then